### PR TITLE
chore(orc8r): upload vagrant box with vagrant cli

### PR DESCRIPTION
**Pull request by maxhbr**
Needs someone to take care of it and to finish this PR.

### CODE REVIEW

File: orc8r/tools/packer/vagrant-box-upload.sh [hunk 1]
```diff
@@ -21,33 +18,10 @@ if [ -z "$VAGRANT_CLOUD_TOKEN" ]; then
...
-else
-    echo -e "\nUpload did not work: $upload_path != $uploaded_path"
-fi
+vagrant cloud auth login --token "$VAGRANT_CLOUD_TOKEN"
+vagrant cloud publish \
+    "${USER}/${BOX}"
+    "$VERSION" \
+    "$BOX_PROVIDER" \
+    "$BOX_FILE" \
+    --release
```
Comment (maxhbr) [inline orc8r/tools/packer/vagrant-box-upload.sh]: this line will automatically release the file directly. Maybe we want to drop that, but that was also the old logic which was just broken.

File: orc8r/tools/packer/vagrant-box-upload.sh [hunk 2]
```diff
@@ -1,15 +1,12 @@
...
-# https://www.vagrantup.com/docs/vagrant-cloud/api.html#creating-a-usable-box-from-scratch
-# https://www.vagrantup.com/docs/vagrant-cloud/boxes/create.html
+set -euo pipefail
 
 BOX_FILE=$1
 
 USER=magmacore
 BOX=$(basename $BOX_FILE | cut -d_ -f1-2 | cut -d. -f1)
-VERSION="1.1.$(date +"%Y%m%d")"
+VERSION="1.2.$(date +"%Y%m%d")"
```
Comment (maxhbr) [inline orc8r/tools/packer/vagrant-box-upload.sh]: I thought we already bumped the version

**Comment by MoritzThomasHuebner** [comment]
My changes, including tests:

https://github.com/maxhbr/magma/pull/1

**Comment by sebathomas** [comment]
This might need a rebase, I think the broken dp integ test is fixed on master.

File: orc8r/tools/packer/vagrant-box-upload.sh (hunk 2 above)
Comment (sebathomas) [reply orc8r/tools/packer/vagrant-box-upload.sh]: And this was changed on master too.

**Comment by maxhbr** [comment]
I will rebase

File: orc8r/tools/packer/vagrant-box-upload.sh [hunk 3]
```diff
@@ -21,33 +18,11 @@ if [ -z "$VAGRANT_CLOUD_TOKEN" ]; then
...
-    echo -en "\nReleasing..."
-    curl --header "Authorization: Bearer $VAGRANT_CLOUD_TOKEN" https://app.vagrantup.com/api/v1/box/$USER/$BOX/version/$VERSION/release --request PUT
-    echo "Done!"
-else
-    echo -e "\nUpload did not work: $upload_path != $uploaded_path"
-fi
+vagrant cloud auth login --token "$VAGRANT_CLOUD_TOKEN"
+vagrant cloud publish \
+    --release \
+    --force \
```
Comment (sebathomas) [inline orc8r/tools/packer/vagrant-box-upload.sh]: I remember we discussed yesterday if `force` was really necessary. Can you remind me why we need it?

Comment (maxhbr) [reply orc8r/tools/packer/vagrant-box-upload.sh]: https://github.com/maxhbr/magma/pull/1#discussion_r957017899 <- that discussion was not resolved

Comment (MoritzThomasHuebner) [reply orc8r/tools/packer/vagrant-box-upload.sh]: I amended the commit to remove the `--force`

**Review by sebathomas — APPROVED** [review]

**Pull request merged**